NightscoutUploadKit/DeviceStatus/LoopEnacted.swift

```diff
@@ -13,20 +13,28 @@ public struct LoopEnacted {
     let duration: NSTimeInterval
     let timestamp: NSDate
     let received: Bool
+    let predBGs: [Double]?
     
-    public init(rate: Double, duration: NSTimeInterval, timestamp: NSDate, received: Bool) {
+    public init(rate: Double, duration: NSTimeInterval, timestamp: NSDate, received: Bool, predBGs: [Double]? = nil) {
```

Comment:
BG units in NS are always mg/dl, and are converted at display time, if an environment variable is set.